.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,13 +4,4 @@ about: Request an enhancement, not a bug fix
 
 ---
 
-## Feature description
-<!-- In as much detail as possible -->
-
-## Image
-<!-- Even a simple sketch of the feature would help -->
-
-## Use case
-<!-- What's your use case for this feature? Even if we can't add it, we might be able to address your case. -->
-
-## Additional information
+Thanks, but we're not adding new features.
```

Comment:
This isn't a secret, from a simple look at the [Changelog](https://wordpress.org/plugins/genesis-custom-blocks/#developers).

This way, we won't disappoint users when they open an issue here and we don't add the feature.